src/components/backend-ai-resource-broker.ts

```diff
@@ -671,6 +646,43 @@ export default class BackendAiResourceBroker extends BackendAIPage {
       this.available_slot = remaining_sg_slot;
       this.used_slot_percent = used_slot_percent;
       this.used_resource_group_slot_percent = used_resource_group_slot_percent;
+
+      const enqueueSession = globalThis.backendaiclient._config.always_enqueue_compute_session === true;
+      const availablePresets = response.presets.map((item) => {
+        if (item.allocatable === true) {
+          for (const [slotKey, slotName] of Object.entries(slotList)) {
+            if (slotKey in item.resource_slots) {
+              item[slotName] = item.resource_slots[slotKey];
+            }
+          }
+        } else if (enqueueSession) {
+          // Even if allocatable is false, if enqueueSession is true,
+          // allocatable is determined based on when no resources are allocated.
+          item.allocatable = true;
+          for (const [slotKey, slotName] of Object.entries(slotList)) {
+            if (slotKey in item.resource_slots && slotName in total_resource_group_slot) {
+              if (item.resource_slots[slotKey] <= total_resource_group_slot[slotName]) {
```

Comment:
When the `slotKey` is `mem`, two comparison slots are in different units: one in bytes and the other in Gbytes. This makes some presets not displayed.

<img width="249" alt="image" src="https://user-images.githubusercontent.com/7539358/209425444-280f5a04-287e-4267-af80-23286545febf.png">

Reply:
Oops, thanks for catching it! I fixed.